Bottom: aeb24633aaa003c8c913afb8e47f284008fa114e
Top:    9d68074f961252da8b6cdeb168132e5d8ea50752
Author: Christopher Voltz <cjunk@voltz.ws>
Date:   2017-05-01 23:45:04 -0500

Refresh of readme

---

diff --git a/README.md b/README.md
index 6238caa..d634551 100644
--- a/README.md
+++ b/README.md
@@ -7,6 +7,8 @@ the original documentation page at: https://www.dokuwiki.org/plugin:tablecalc.
 Forked from https://github.com/pirati-cz/tablecalc. Adds the following
 functions:
 
-  | ''tag(name)''     | Associate the current row and column with a name |
-  | ''tag_col(name)'' | Return the column for the specified tag          |
-  | ''tag_row(name)'' | Return the row for the specified tag             |
+  | Function        | Description                                      |
+  | --------------- | ------------------------------------------------ |
+  | `tag(name)`     | Associate the current row and column with a name |
+  | `tag_col(name)` | Return the column for the specified tag          |
+  | `tag_row(name)` | Return the row for the specified tag             |
